CourseGrab/Info.plist

```diff
@@ -16,6 +16,17 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.894460636145-mulqctlklp1alf73u43kbdm5n7vulqga</string>
```

Comment:
What should we do about this file if it also has the key? Add it to gitignore?

Reply:
You could create a user defined variable under `Build Settings` and reference its value in `Info.plist` by doing `$(VAR_NAME)`. You can take a look here for reference (https://stackoverflow.com/questions/10854156/how-to-create-variables-for-use-in-xcode-build-settings-values)

Reply:
Hmm, not sure. For Pollo, we also have the key in the `Info.plist` and it is _not_ in the `.gitignore`. I think it is fine unless someone else thinks otherwise.

Reply:
Would it work if you moved it to the `Secrets/Keys.plist`?